registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -102,6 +102,7 @@
     "sha256:d115f04597212b26bf2f17e9b056821f86a9cc5ad5bd4dcb7bd27c5586df7f35": ["v1.11.0-rc.0"]
     "sha256:e2d630371969d670bcf96b636f030bf07ebad61f8f06a93f2fe01dab6eee478f": ["v1.11.0-rc.1"]
     "sha256:54fde214e6ccadf40ccd788e1a192d078bda981d45e7ec8af460bca54915790e": ["v1.11.1"]
+    "sha256:134a8f294ee7c9d18f7d0d0eb62ac81e93dd6aa694d51197080fe2ec9ea8538a": ["v1.11.2"]
```

Comment:
SHA `134a8f294ee7` matches cluster-api-controller v1.11.2 in the [staging repo](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/cluster-api-controller@sha256:134a8f294ee7c9d18f7d0d0eb62ac81e93dd6aa694d51197080fe2ec9ea8538a/details).